Tutorial needs to be updated.

Vikramkulkarni01 opened the pull request:
changed screenshot

Vikramkulkarni01 commented:
> Every note should not say IMPORTANT -- only things that if you miss and likely to miss that nothing will work. Explanatory text should not be labeled IMPORTANT. Probably only one note should be labeled important, perhaps.
@thecodester this is important, since they are mixed behaviors for old and new accounts. Only new account users should follow this. So, please include

thecodester commented:
All the following relates to cp-starter-integration-cpi-onboard-subscribe

Step 2 starts with a note called Important. Just write this as plain text.

Step 2, substep 4 -- again, I don't find this more important than the actual steps they need to follow.

Step 2, substep 5 -- this is definitely not important, it is just additional information.

Step 3 starts with important note  that is definitely not important

Step 4 at the end, this could be considered important, so OK. No space between the 2 sentences of this note.

Step 5 starts with a header "Details" -- please remove

Vikramkulkarni01 commented:
Hi @thecodester, 

Please find my response inline. 

Step 2 starts with a note called Important. Just write this as plain text.
--> I added as a plain text.

Step 2, substep 4 -- again, I don't find this more important than the actual steps they need to follow.
--> This is required since many customers faced with multiple identifiers. 

Step 2, substep 5 -- this is definitely not important, it is just additional information.
--> I removed this. 

Step 3 starts with an important note that is definitely not important
--> I converted from important to plain texts as its important due to predefined templates. 


Step 4 at the end, this could be considered important, so OK. No space between the 2 sentences of this note.
--> In atom editor I can see the space. However, I have given an additional one. 

Step 5 starts with a header "Details" -- please remove
--> This has been provided by Info architect as a part of revamping onboarding experience. So please retain it. 


Kindly request you to merge the code. As the customers are waiting for some updates. 

I appreciate your help.
